python/tests/sbp/client/test_logger.py

```diff
@@ -133,9 +137,9 @@ def test_rolling_json_log():
       i = 0
       with open(tf.name, 'r') as infile:
         with JSONLogIterator(infile) as log:
-          for msg, metadata in log.next():
+          for msg, metadata in next(log):
             assert isinstance(msg, MsgPrintDep)
-            assert msg.text == "abc\n"
+            assert msg.text == b"abc\n"
             i += 1
       assert i > 0
       assert i <= len(msgs)
```

Comment:
long term I would like to see a performance test or duration test here to ensure that changes are making python logging more performant or at least not making it slower.